.github/workflows/docker-be.yml

```
@@ -0,0 +1,92 @@
+name: BE Docker CI
+
+on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)